Src/PCompiler/PCommandLine/Parser/ParsePProjectFile.cs

```diff
@@ -183,43 +208,48 @@ private string GetOutputDirectoryName(FileInfo fullPathName)
                 return Directory.GetCurrentDirectory();
         }
 
-        private void GetTargetLanguage(FileInfo fullPathName, ref CompilerOutput outputLanguage, ref bool generateSourceMaps)
+        private IDictionary<string, CompilerOutput> GetTargetLanguages(FileInfo fullPathName)
```

Comment:
Why is the return value a dictionary and not a list?

Reply:
Update to list via [dd21650](https://github.com/p-org/P/pull/660/commits/dd21650774c9819d57d32cb8ad91eda2a1b1ac03)